### ISSUE TITLE
how I can fix vue warn?

### ISSUE THREAD

**Issue by codegor** [formly-js/vue-formly#34]
Hi. I use the template definition:

`<formly-form :form="form" :model="model" :fields="conf1"></formly-form>`

and when I run my app I see this Vue warn on the chrome console:

```
[Vue warn]: The computed property "fields" is already defined as a prop.

found in

<FormlyForm>

```
How I can fix this warn?

**Comment by matt-sanders** [comment]
hmm, I haven't seen this error before, I'll try and take a look soon and let you know

**Comment by codegor** [comment]
Yes, it is that plugin - https://github.com/baianat/vee-validate/blob/master/src/mixin.js

**Comment by codegor** [comment]
may be you can change name of var fields or cretae new second var, so that the backward compatibility for existing users remains
second var it is one and the same with var fields just one more name to eliminate the conflict

**Comment by codegor** [comment]
I have created bug for vue-validate https://github.com/baianat/vee-validate/issues/1302

**Comment by codegor** [comment]
Can you advice how i can use your plugin with vee-validate?

**Comment by codegor** [comment]
I like this code in your doc:
`this.$refs.loginForm.validate()
                .then(()=>{
                    if ( !this.form.$valid ) return;
                    // submit form
                })
                .catch((e)=>{
                    // do something with the error
                });`
but how i can set rules that after i run validate() all fields will be validated and if all ok will run then()

**Comment by matt-sanders** [comment]
Great that you've found the issue. I don't know if those two would really be compatible together. That plugin runs validation on an input, which vue formly already does. It would seem that by adding vee-validate you're doubling up on functionality. 

As per the docs, `validate()` will return a promise. When it's returned you just need to check the `this.form.$valid` status and then you can either return and do nothing if it's invalid or continue. `then()` always runs, because the promise always resolves, ( unless there is a script error of some sort ). This would then indicate that the `validate()` function successfully ran each validation E.g.
```
this.$refs.loginForm.validate()
   .then( () => {
      // if the form is invalid, then return
      if ( !this.form.$valid ) return;

      // otherwise, submit the form or something
      yourLogicHere();
   })
